synapseAnnotations/data/cancer.json

```diff
@@ -232,6 +232,11 @@
         "description": "A neoplasm that arises from SCHWANN CELLS of the cranial, peripheral, and autonomic nerves. ",
         "source": "https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0000693"
       },
+      {
+        "value": "Lymphoblastic Lymphoma",
+        "description":"A lymphoma composed of immature small to medium-sized precursor lymphoid cells (lymphoblasts). It includes the B- and T-cell lymphoblastic lymphoma.",
+        "source": "https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&version=18.06d&ns=ncit&code=C9360&key=n4324201&b=1&n=null"
```

Comment:
Would suggest instead using the OBO link for source: http://purl.obolibrary.org/obo/NCIT_C9360